src/test/fuzz/versionbits.cpp

```diff
@@ -343,3 +349,4 @@ void test_one_input(const std::vector<uint8_t>& buffer)
         }
     }
 }
+} // namespace
```

Comment:
This `} // namespace` line needs to be removed for the backport.

Reply:
Correct. Fixed.